src/app/search/search-filter.ts

```diff
@@ -156,18 +160,18 @@ function makeSearchFilterFactory(
             } catch (e) {
               // TODO: mark invalid - fill out what didn't make sense and where it was in the string
               errorLog('search', 'Invalid query term', filterName, filterValue, e);
-              return () => true;
+              return undefined;
             }
           }
 
           // TODO: mark invalid - fill out what didn't make sense and where it was in the string
-          return () => true;
+          return undefined;
         }
         case 'noop':
-          return () => true;
+          return undefined;
       }
     };
 
-    return transformAST(parsedQuery);
+    return transformAST(parsedQuery) ?? (() => true);
```

Comment:
This is the existing behavior -- if a query has not a single valid filter, we match everything. Maybe `false` would be better here?

Reply:
My reasoning here was that if you don't have a valid filter, we shouldn't just grey out every single item.